app/components/pages/PlatformAppStore.vue.ts

```diff
@@ -6,26 +6,49 @@ import { GuestApiService } from 'services/guest-api';
 import { I18nService } from 'services/i18n';
 import electron from 'electron';
 import { PlatformAppsService, EAppPageSlot } from 'services/platform-apps';
+import { PlatformAppStoreService } from 'services/platform-app-store';
+import Utils from 'services/utils';
 
 @Component({})
 export default class PlatformAppStore extends Vue {
   @Inject() userService: UserService;
   @Inject() platformAppsService: PlatformAppsService;
+  @Inject() platformAppStoreService: PlatformAppStoreService;
   @Inject() guestApiService: GuestApiService;
   @Inject() i18nService: I18nService;
 
   $refs: {
-    appStore: Electron.WebviewTag;
+    appStoreWebview: Electron.WebviewTag;
   };
 
   mounted() {
-    this.$refs.appStore.addEventListener('dom-ready', () => {
-      // if (this.platformAppsService.state.devMode) {
-      //   this.$refs.appStore.openDevTools();
-      // }
+    this.$refs.appStoreWebview.addEventListener('dom-ready', () => {
+      if (Utils.isDevMode()) {
+        this.$refs.appStoreWebview.openDevTools();
+      }
+      this.guestApiService.exposeApi(
+        this.$refs.appStoreWebview.getWebContents().id,
+        {
+          onInstallApp: this.onInstallAppHandler,
```

Comment:
A function that starts with `on` generally implies that you are binding a callback for an event rather than performing an action.  I would expect these functions to be called something like:
`refreshApps` or `refreshAndLoadApps` maybe
`openLinkInBrowser`
`onPaypalSuccess` or smiliar.

Do those sound reasonable?

Reply:
yeah makes sense, dont know what I was thinking. Thanks Andy